distribution/docs/src/main/resources/content/_tables/Csw_Connected_Source.adoc

```diff
@@ -5,7 +5,7 @@
 :application: ${ddf-catalog}
 :summary: CSW Connected Source.
 
-.[[Csw_Connected_Source]]CSW Connected Source
+.[[_Csw_Connected_Source]]CSW Connected Source
```

Comment:
❓ I'll leave one final comment on this topic, but a lot of places look like we refer to these sections with prefixes, but don't use the same prefix here. Should we be using prefixes in these sections?

Reply:
see previous comment